javascript/codefight/addBorder.js

```diff
@@ -1,6 +1,6 @@
 export const addBorder = (picture) => {
-    let outPut = [];
-    let firstEl = picture[0].length;
+    const outPut = [];
```

Comment:
This array is initially empty, and then filled. The value of a constant cannot change through re-assignment, and it can't be redeclared.

Reply:
In JS you can declare array with const and push items in it. It will only throw an error if you will try to re-assign variable.

Reply:
Really, thanks! )